@ecomplus/storefront-components/src/scss/TheProduct.scss

```
@@ -130,6 +130,27 @@
     }
   }
 
+  .sale-timer {
+    width: 100%;
+    max-width: 300px;
+    background-color: var(--primary);
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    padding: 5px 10px;
+    color: #ffffff;
```

```suggestion
    color: var(--secondary-light-yiq);
```

Nem sempre as cores da marca são escuras, portanto nem sempre branco é um bom foreground, a variável YIQ respectiva sempre traz a cor com contraste

---

ok